lib/ex_doc/formatter/html.ex

```diff
@@ -52,6 +55,37 @@ defmodule ExDoc.Formatter.HTML do
     config.output |> Path.join("index.html") |> Path.relative_to_cwd()
   end
 
+  defp dep_completions(app, config) do
+    # TODO: proper way of building this path
+    ebin_dir =
+      Path.join([File.cwd!(), "_build", to_string(Mix.env()), "lib", to_string(app), "ebin"])
```

Comment:
Is there a better way to access the `build` directory?

Reply:
```elixir
iex> Mix.Project.build_path()
"/Users/wojtek/src/ex_doc/_build/dev"
```